test/test-filter-process.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+. "test/testlib.sh"
+
+if [ "$GIT_LFS_USE_LEGACY_FILTER" == "1" ]; then
+  echo "skip: $0 (filter process disabled)"
+  exit
+fi
+
+# HACK(taylor): git uses ".g<hash>" in the version name to signal that it is
+# from the "next" branch, which is the only (current) version of Git that has
+# support for the filter protocol.
```

Comment:
FYI that `g` prefix is from [`git describe`](https://git-scm.com/docs/git-describe). 

> The "g" prefix stands for "git" and is used to allow describing the version of a software depending on the SCM the software is managed with.

Sorry if you already know that, the comment made it seem like you didn't.  It's not something I've ever encountered in normal git use, just github feature dev :)